Gemfile.lock

```diff
@@ -277,34 +272,34 @@ GEM
       activerecord (>= 5.0)
     psych (3.1.0)
     public_suffix (4.0.1)
-    puma (4.2.0)
+    puma (4.3.1)
```

Comment:
### [puma](http://puma.io)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 4.2.0 → 4.3.1

[_[change-log](https://github.com/puma/puma/blob/master/History.md), ~~source-code~~_]

:tada: Patched vulnerabilities:

- [CVE-2019-16770](https://nvd.nist.gov/vuln/detail/CVE-2019-16770)
  **Keepalive thread overload/DoS in puma**

  CVSS V2: [6.8 high](https://nvd.nist.gov/vuln-metrics/cvss/v2-calculator?name=CVE-2019-16770)
  URL: https://github.com/puma/puma/security/advisories/GHSA-7xx3-m584-x994

  A poorly-behaved client could use keepalive requests to monopolize Puma's reactor and create a denial of service attack.  If more keepalive connections to Puma are opened than there are threads available, additional connections will wait permanently if the attacker sends requests frequently enough.